app/pages/events/$id.mjs

```diff
@@ -3,9 +3,59 @@ import { marked } from "marked"
 export default function ({ html, state = {} }) {
   let { store = {} } = state
   let event = store.events[0]
-  let { id, title, sponsors, talks, description } = event
+  let display = store.display
+  let { id, title, sponsors, talks, description, date } = event
   let hasTalks = talks && talks.length > 0
   let hasSponsors = sponsors && sponsors.length > 0
+  if (display === "email") {
```

Comment:
The file is a liiiittle bit big, and it might be cleaner to extract the email template into its own component. I decided to keep it in one file because the null checks would be duplicated and the code for the regular page (the nicely rendered one) is short